sync.go

```diff
@@ -417,9 +345,9 @@ func (lw *LibWallet) RescanBlocks() error {
 			estimatedTotalRescanTime := int64(math.Round(float64(elapsedRescanTime) / rescanRate))
 			rescanProgressReport.RescanTimeRemaining = estimatedTotalRescanTime - elapsedRescanTime
 
-			for _, syncProgressListener := range lw.syncProgressListeners {
-				syncProgressListener.OnHeadersRescanProgress(rescanProgressReport)
-			}
+			// for _, syncProgressListener := range mw.syncProgressListeners {
```

Comment:
Was this commented out for test purposes?

Reply:
Commented out to be considered later. New design doesn't come with a rescan function and might consider removing it.